apps/bfd-server/dev/bfd-data-dictionary/data/V2/0419_eob_resource_last_updated_date.json

```diff
@@ -37,7 +37,7 @@
             "additional": [],
             "derived": "",
             "note": "Format of the timestamp with time zone, of this field is as follows:  YYYY-MM-DDTHH24:MI:SS.TIMEZONE",
-            "example": "[\\n  \"IGNORED_FIELD\"\\n]"
+            "example": "[\\n  {\\n    \"lastUpdated\": \"2021-05-15T08:22:38.251+00:00\",\\n    \"profile\": [\\n      \"http://hl7.org/fhir/us/carin-bb/StructureDefinition/C4BB-ExplanationOfBenefit-Inpatient-Institutional\"\\n    ]\\n  }\\n]"
```

Comment:
Isn't this timezone value diff from expected? Don't think we need ms here.

Reply:
Spec says it's an instant which includes ms, YYYY-MM-DDThh:mm:ss.sss+zz:zz, so maybe the note is wrong?

Reply:
Yea, not crazy about some of the FHIR stuff; this one really irks me, especially with respect to claims data...we'd never have ms granularity! My gripe was the test value does not reflect the comment (format) or vice-versa; I'll mark as resolved b